src/rma/js/components/destruction-list/zaken-table.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { CheckboxInput } from "./inputs";
+
+
+function displayZaaktype (zaaktype) {
+    return `${zaaktype.omschrijving} (${zaaktype.versiedatum}}`;
+}
+
+
+function ZakenTable(props) {
+    const { zaken, isLoaded, error, checkboxes, setCheckboxes } = props;
+
+    const [selectAll, setSelectAll] = useState(false);
+
+    if (error) {
```

Comment:
_Could_ be relevant: https://reactjs.org/docs/error-boundaries.html

Reply:
afaik error boundaries are implemented now only for classes (https://reactjs.org/docs/hooks-faq.html#do-hooks-cover-all-use-cases-for-classes)

Reply:
check :+1: